quickstatements/README.md

```diff
@@ -0,0 +1,88 @@
+# Quickstatements docker image
+
+Quickstatements2 as seen at  [https://tools.wmflabs.org/quickstatements/](https://tools.wmflabs.org/quickstatements/)
+
+### Tags
+Image name                          | Parent image             | Quickstatments version
+-------------------------------     | ------------------------ | --------------
+`wikibase/quickstatements` : `latest` | [php:5.6-apache](https://hub.docker.com/_/php/) | master
+
+### Environment variables
+
+Variable                  | Default              | Description
+--------------------------|----------------------|-----------
+`WIKIBASE_HOST`           | NONE     | Host of wikibase instance as seen by QS container
+`WB_PUBLIC_HOST_AND_PORT` | NONE     | Host and port of wikibase as seen by the user's browser
+`QS_PUBLIC_HOST_AND_PORT` | NONE     | Host and port of QS as seen by the user's browser
+`OAUTH_CONSUMER_KEY`      | NONE     | OAuth consumer key (obtained from wikibase)
+`OAUTH_CONSUMER_SECRET`   | NONE     | OAuth consumer key (obtained from wikibase)
+
```

Comment:
There are quite a few files that people might want to override in this image, it would be good to document where they are as in other readmes.